server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -118,17 +131,25 @@
     }
 
     try {
-      const { headings, tableData } = await standardReportDAO.post(app, req, reportType.apiKey, config);
+      const { headings, tableData } = await (reportType.bespokeReport.dao
```

Comment:
update to:
`reportType.bespokeReport && reportType.bespokeReport.dao`
otherwise it breaks standard reports that arent bespoke

Reply:
Weird, I tested them 😬 
There probably should at least be some null coalescing here, I'll make a change to make it obvious what's going on, because it isn't to me.

Reply:
Ok yeah I didn't test them very well :D